### PR TITLE
[DGPFUS-37] Database Design For A Transportation Company

### PR DESCRIPTION
**Pull request by ayodejimb**
In this task, I designed a database for a transportation company based on the given criteria. Kindly check it out. Thanks.

### CODE REVIEW

File: submissions/mubarakOlaoluwa/Database/transportCompany_database.sql [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+CREATE DATABASE transport;
+
+/*Selecting the database after creation*/
+USE transport;
+
+/*Creating the first table (passengers)*/
+CREATE TABLE passengers (
+	id INT NOT NULL AUTO_INCREMENT,
+    full_name VARCHAR(250) NOT NULL,
+    sex VARCHAR(1) NOT NULL,
```
Comment (ToluEnny24) [inline submissions/mubarakOlaoluwa/Database/transportCompany_database.sql:11]: change this to accommodate more characters.

File: submissions/mubarakOlaoluwa/Database/transportCompany_database.sql [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+	id INT NOT NULL AUTO_INCREMENT,
+    class INT NOT NULL,
+    ticket_number VARCHAR(20) NOT NULL,
+    fare INT NOT NULL,
+    cabin VARCHAR (10),
+    parents_or_children INT NOT NULL,
+    siblings_or_spouses INT NOT NULL,
+    embarkation VARCHAR (50),
+    PRIMARY KEY (id),
+    FOREIGN KEY (class) REFERENCES passengers (id)
```
Comment (ToluEnny24) [inline submissions/mubarakOlaoluwa/Database/transportCompany_database.sql:27]: the class is a different detail different from the passengers id which is the link between this table and the passenger's table.

File: submissions/mubarakOlaoluwa/Database/transportCompany_database.sql [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    PRIMARY KEY (id),
+    FOREIGN KEY (class) REFERENCES passengers (id) 
+);
+
+/*Creating the third table (accidents)*/
+CREATE TABLE accidents(
+	id INT NOT NULL AUTO_INCREMENT,
+    survived VARCHAR(3) NOT NULL, 
+    PRIMARY KEY (id),
+    FOREIGN KEY (id) REFERENCES trips (id)
```
Comment (ToluEnny24) [inline submissions/mubarakOlaoluwa/Database/transportCompany_database.sql:35]: id can not be foreign and Primary key. Rather you should have passengers-id which references the passengers table.

**Review by ToluEnny24 — CHANGES_REQUESTED** [review]
Go through each of the comments and effect the changes. @ayodejimb

**Comment by ayodejimb** [comment]
Thank you. I have made the corrections. Kindly review. Thanks!!

**Pull request merged**